### PR TITLE
fix(get_grafana_annotations): make the limit much larger

### PR DESCRIPTION
**Pull request by fruch**
Now the default limit in grafana API is 100, which mean we extract only 100 of the last events
and when we use `hydra investigate show-monitor` we have last 100 events available.

In longer runs like 48h and more, this isn't helpful, and we are the context like that.

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/utils/monitorstack.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def get_monitoring_stack_scylla_version(monitoring_stack_dir):
         with open(os.path.join(monitoring_stack_dir, 'monitor_version'), 'r') as f:  # pylint: disable=invalid-name
             versions = f.read().strip()
         monitoring_version, scylla_version = versions.split(':')
-        return monitoring_version, scylla_version
+        return monitoring_version, "all"
```
Comment (bentsi) [inline sdcm/utils/monitorstack.py:214]: how is this related to this PR? What is the purpose/reason for this change?

Comment (fruch) [reply sdcm/utils/monitorstack.py:214]: It's not, I've noticed that when we use `show-monitor` we don't have the other dashboards, since scylla version comes out as 4.4 on master, and we don't have a versions.

I think we could just put all of the versions available in scylla-monitoring, in case we need them.

Comment (fruch) [reply sdcm/utils/monitorstack.py:214]: I'll take it to a different PR

Comment (fruch) [reply sdcm/utils/monitorstack.py]: after a closer look, 
the problem was in the way we handle the version, took it to this PR:
https://github.com/scylladb/scylla-cluster-tests/pull/2847

**Review by bentsi — APPROVED** [review]

**Comment by fruch** [comment]
thanks @bentsi
